modules/fontconfig/meta.nix

```
@@ -0,0 +1,16 @@
+{ lib, ... }:
+{
+  name = "Fontconfig";
+  homepage = "https://fontconfig.org";
+  maintainers = [ lib.maintainers.mightyiam ];
+  description = ''
+    On NixOS:
+
+    - does _not_ enable Fontconfig
+    - adds the Stylix fonts with default ordering to `fonts.fontconfig.defaultFonts`
```

Nit:

I find this sentence unclear. What is meant by "default ordering to" in this context?

Should it be something like:
```suggestion
    - configures the Stylix fonts in the `fonts.fontconfig.defaultFonts` options
```
?